src/components/QuranReader/TranslationView/TranslatioViewCellSkeleton.tsx

```diff
@@ -36,50 +35,59 @@ const TranslationViewCellSkeleton: React.FC<Props> = ({ hasActionMenuItems = tru
   return (
     <div className={classNames(cellStyles.cellContainer, skeletonStyles.cellContainer)}>
       <div className={cellStyles.actionContainer}>
-        <Skeleton className={cellStyles.actionContainerLeft}>
-          <Button size={ButtonSize.Small}>{VERSE_KEY_SAMPLE}</Button>
-        </Skeleton>
+        <div className={cellStyles.actionContainerLeft}>
+          <Skeleton className={skeletonStyles.actionItem}>
+            <Button size={ButtonSize.Small} />
+          </Skeleton>
+          <Skeleton className={skeletonStyles.actionItem}>
+            <Button size={ButtonSize.Small} />
+          </Skeleton>
+          <Skeleton className={skeletonStyles.actionItem}>
+            <Button size={ButtonSize.Small} />
+          </Skeleton>
+          <Skeleton className={skeletonStyles.actionItem}>
+            <Button size={ButtonSize.Small} />
+          </Skeleton>
```

Comment:
nit: maybe a loop will be better in-case we will change the number of items in the future

Reply:
yes, I agree 👍